scenarios/covid/README.md

```diff
@@ -130,7 +130,7 @@ cd scenarios/covid/data
 
 ### Sign and Register Contract
 
-Next, follow instructions [here](./../../external/contract-ledger/README.md) to sign and register a contract with the contract service. You can either deploy your own contract service or use a test contract service hosted at ```https://contract-service.westeurope.cloudapp.azure.com:8000```. The registered contract must contain references to the datasets with matching names, keyIDs and Azure Key Vault endpoints used in this sample. A sample contract template for this scenario is provided [here](./contract/contract.json). After updating, signing and registering the contract, retain the contract service URL and sequence number of the contract for the rest of this sample. 
+Next, follow instructions [here](https://github.com/kapilvgit/contract-ledger/blob/675003b83211e6d3d2c15864523bf875e0172cba/demo/contract/README.md) to sign and register a contract with the contract service. You can either deploy your own contract service or use a test contract service hosted at ```https://contract-service.eastus.cloudapp.azure.com:8000/```. The registered contract must contain references to the datasets with matching names, keyIDs and Azure Key Vault endpoints used in this sample. A sample contract template for this scenario is provided [here](./contract/contract.json). After updating, signing and registering the contract, retain the contract service URL and sequence number of the contract for the rest of this sample. 
```

Comment:
Why this specific commit?

Reply:
1. link at the top is not working. So, I corrected that. 
2. I have created a new contract service at eastus and hope to continue to host, as we have ispirt account to host this now, so that this can be leveraged for any one wants to test the code.